packages/credit-card-integration/src/create-credit-card-payment-strategy.ts

```diff
@@ -0,0 +1,19 @@
+import {
+    PaymentStrategyFactory,
+    toResolvableModule,
+} from "@bigcommerce/checkout-sdk/payment-integration-api";
+
+import CreditCardPaymentStrategy from "./credit-card-payment-strategy";
+
+const createCreditCardPaymentStrategy: PaymentStrategyFactory<
+    CreditCardPaymentStrategy
+> = (paymentIntegrationService) => {
+
+    return new CreditCardPaymentStrategy(
+        paymentIntegrationService,
+    );
+};
+
+export default toResolvableModule(createCreditCardPaymentStrategy, [
+    { default: true },
```

Comment:
I think a bit more work needs to be done if we are using this approach. Right now, we use `PaymentStrategyRegistryV2` to resolve first, and then fall back to `PaymentStrategyRegistry`, e.g.: https://github.com/bigcommerce/checkout-sdk-js/blob/aaac6a157ec6783782460ca1851859d245aedff2/packages/core/src/payment/payment-strategy-action-creator.ts#L92-L103

So if there's a default strategy registered with the V2 registry, we'll not able to use V1 as a fallback. Perhaps we can flip it around so we resolve with V1 registry first and then V2 registry, as V1 registry no longer has a default strategy.

PS: I still prefer to list out the methods explicitly but I understand it's a lot more work to figure out which one to list out so I guess `default` is a good compromise for now.